src/components/Tabber/Tabber.tsx

```diff
@@ -1,8 +1,9 @@
-import { ReactNode, useState } from 'react';
+﻿import React, { useState } from 'react';
```

Comment:
se nao me engano o react ja vem importado por padrao hoje em dia e nao é necessario mais importar ele, da uma olhada se realmente precisa importar o react inteiro aqui por favor

Reply:
precisa pq to usando um reactcomponent, toda vez que removo da erro.

Sobre o teste irei fazer o teste, mas nem consigo em pensar em um bom, já que o Alvaro está tendo que refazer o modulo. Então vai sair um bem simples